.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -10,14 +10,17 @@ body:
       label: Search before asking
       description: |-
         Please search [issues](https://github.com/FasterXML/jackson-module-kotlin/issues) to check if your issue has already been reported.
+        Check [all value class labeled issues](https://github.com/FasterXML/jackson-module-kotlin/issues?q=is%3Aopen+is%3Aissue+label%3A%22value+class%22), especially if the problem is related to a value class.
```

Comment:
The `value class` behaved differently than expected and was difficult to search by the user.
Therefore, I have added a statement that prompts the user to check all labeled issues.